src/app/header/header.component.html

```diff
@@ -1,74 +1,157 @@
-<nav class="navbar navbar-default navbar-pf" role="navigation" *ngIf="loggedIn">
-    <div class="navbar-header"></div>
+<nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="loggedIn">
```

Comment:
yes ... once could write drop down menu as a sub component ... but that would prob be a premature optimization at this point ... so initially it's simpler and allows for standard usage of patternfly bootstrap menu without forcing a round peg into a square hole and writing duplicate script code. ... patternfly got a certain paradigm with the second level menus that makes it easier and simpler to do it this way